_sass/fabric.scss

```diff
@@ -1,3 +1,8 @@
+@font-face {
```

Comment:
This will include this font all pages even if its not used.

Reply:
The font won't be requested by the browser unless there is an element that uses it
https://stackoverflow.com/questions/2756575/drawing-text-to-canvas-with-font-face-does-not-work-at-the-first-time

Reply:
Can it be added to the css just for the template generator page (bottom of Template.svelte), there should be no need for it on the main page.